spec/acceptance/nrpe_spec.rb

```diff
@@ -40,15 +39,9 @@
   end
 
   describe 'with custom server port' do
-    it 'updates without errors' do
-      pp = <<-EOS
-      class { 'nrpe':
-        server_port => 9999,
-      }
-      EOS
-      apply_manifest(pp, catch_failures: true)
-      apply_manifest(pp, catch_changes: true)
-    end
+    let(:pp) { 'class { \'nrpe\': server_port => 9999 }' }
```

Comment:
If i understand well block syntax with `let` is better.
But to permit easier reading, i prefer to not use a one line syntax when back slash is needed.

Can you change this keeping the block syntax but without back slash ?

Reply:
```ruby
let(:pp) { "class { 'nrpe': server_port => 9999 }" }
```
?

Reply:
@Dan33l Fixed